### ISSUE TITLE
 新版本报 HTTPSConnectionPool 

### ISSUE THREAD

**Issue by Lofalt** [chinshin/CQBot_hzx#20]
2019-04-17 11:02:15,178 - WARNING - 
error when getRoomMsg
HTTPSConnectionPool(host='pocketapi.48.cn', port=443): Max retries exceeded with url: /im/api/v1/chatroom/msg/list/homeowner (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x00000000049DDD30>, 'Connection to pocketapi.48.cn timed out. (connect timeout=5)'))

TOKEN好像也获取不了.0.

**Comment by chinshin** [comment]
是的 应该是IP被ban，口袋更新的时候你没关插件吧。https://pocketapi.48.cn/ 这个能看到Welcome to nginx!就是没被封，看不到就是被封了

**Comment by Lofalt** [comment]
的确是被ban了 请教一下我用的阿里云服务器 好像不可以改IP 是不是相当于报废了T_T

**Comment by chinshin** [comment]
开工单换IP

**Comment by Lofalt** [comment]
好的 我试试~谢谢

**Comment by Heheda48** [comment]
我也遇到了同样的问题，申请工单不让更改IP地址，请问还有别的解决方法吗？

**Comment by Lofalt** [comment]
客服让我 降配到0M 然后买一个弹性公网IP绑定上 不过我没试 应该可行？
